pubspec.lock

```diff
@@ -35,7 +35,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.5.0-nullsafety"
+    version: "2.4.2"
```

Comment:
Your packages are out of date. Make sure you're on the beta channel (`flutter channel beta`) and run `flutter pub get`.